Rakefile

```diff
@@ -145,4 +145,56 @@ namespace :site do
       abort "You seem to have misplaced your History.markdown file. I can haz?"
     end
   end
+
+  desc "generate importer-dependencies data file"
+  task :generate_dependency_data do
+    require "jekyll-import"
+
+    # Monkey-patch to alter behavior just for this task.
+    module JekyllImport
+      def self.require_with_fallback(gems)
+        Array(gems).flatten
+      end
+    end
+
+    data_dir   = File.expand_path("docs/_data", __dir__)
+    data_file  = File.join(data_dir, "importer_dependencies.yml")
+    importers  = JekyllImport::Importer.subclasses
+    label_size = importers.map { |klass| klass.to_s.split("::").last.length }.max + 1
+
+    # available as part of Ruby Stdlib
+    std_lib = %w(csv date fileutils json open-uri rubygems time uri yaml)
+
+    FileUtils.mkdir_p data_dir
+    data = {}
+
+    puts "\nLogging importer dependencies..\n\n"
+    importers.each do |importer|
+      doc_name = importer.to_s.split("::").last.downcase
+      deps = importer.require_deps - std_lib
+      next if deps.empty?
+
+      deps.map! do |dep|
+        if dep.start_with?("active_support")
+          "activesupport"
+        elsif dep == "net/http"
+          "net-http"
```

Comment:
Isn't this part of stdlib too? 🤔

Reply:
Ah yes it is..
I misread the [release post for Ruby 3.1.0](https://www.ruby-lang.org/en/news/2021/12/25/ruby-3-1-0-released/).

Will update.